### ISSUE TITLE
PSequel crashes at startup with MacPorts OpenSSL installed

### ISSUE THREAD

**Issue by psylum** [psequel/psequel-issues#54]
Downloaded the 1.2.1beta as referenced in the comment on #25.
Copied over previously installed version into /Applications/.
Opened application (hit ok about downloaded binary).

Crash Report:
```
Process:               PSequel [22962]
Path:                  /Applications/PSequel.app/Contents/MacOS/PSequel
Identifier:            com.psequel.PSequel
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           PSequel [22962]
User ID:               501

Date/Time:             2015-07-13 14:35:42.518 +1000
OS Version:            Mac OS X 10.10.4 (14E46)
Report Version:        11
Anonymous UUID:        B0BF13BF-DD82-D2E8-5235-0D71C7464753

Sleep/Wake UUID:       B25862E7-DDC4-427A-A419-6A2AAF98FB4A

Time Awake Since Boot: 210000 seconds
Time Since Wake:       6300 seconds

Crashed Thread:        0

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/opt/openssl/lib/libssl.1.0.0.dylib
  Referenced from: /Applications/PSequel.app/Contents/Frameworks/libpq.5.dylib
  Reason: image not found

.. binary list elided, can provide specifics if need be ..

Model: MacBookPro11,2, BootROM MBP112.0138.B15, 4 processors, Intel Core i7, 2 GHz, 8 GB, SMC 2.18f15
Graphics: Intel Iris Pro, Intel Iris Pro, Built-In
Memory Module: BANK 0/DIMM0, 4 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343531533641465238412D50422020
Memory Module: BANK 1/DIMM0, 4 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343531533641465238412D50422020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x134), Broadcom BCM43xx 1.0 (7.15.166.24.3)
Bluetooth: Version 4.3.5f8 15969, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
Serial ATA Device: APPLE SSD SM0512F, 500.28 GB
USB Device: Internal Memory Card Reader
USB Device: BRCM20702 Hub
USB Device: Bluetooth USB Host Controller
USB Device: Apple Internal Keyboard / Trackpad
Thunderbolt Bus: MacBook Pro, Apple Inc., 17.1
```

MacPorts installed list:
```
The following ports are currently installed:
  bash @4.3.33_0 (active)
  bash-completion @2.1_8 (active)
  boost @1.56.0_2+no_single+no_static+python27 (active)
  bzip2 @1.0.6_0 (active)
  cmake @3.0.2_1 (active)
  curl @7.41.0_0+ssl (active)
  curl-ca-bundle @7.41.0_0 (active)
  db48 @4.8.30_4 (active)
  db_select @0.1_2 (active)
  doxygen @1.8.7_0 (active)
  expat @2.1.0_0 (active)
  gdbm @1.11_1 (active)
  gettext @0.19.4_0 (active)
  git @2.3.4_0+bash_completion+credential_osxkeychain+doc+pcre+perl5_16+python27 (active)
  glfw @20141105_0+docs (active)
  icu @54.1_0 (active)
  libarchive @3.1.2_0 (active)
  libedit @20140620-3.1_0 (active)
  libiconv @1.14_0 (active)
  libidn @1.29_0 (active)
  libpng @1.6.15_0 (active)
  libxml2 @2.9.2_1 (active)
  lzo2 @2.08_0 (active)
  mercurial @3.2.3_0 (active)
  ncurses @5.9_2 (active)
  nodejs @0.10.35_1+python27+ssl (active)
  npm @2.1.17_0 (active)
  openssl @1.0.2a_0 (active)
  p5.16-authen-sasl @2.160.0_0 (active)
  p5.16-cpan-meta @2.143.240_0 (active)
  p5.16-cpan-meta-requirements @2.132.0_0 (active)
  p5.16-cpan-meta-yaml @0.12.0_0 (active)
  p5.16-digest-hmac @1.30.0_0 (active)
  p5.16-digest-sha1 @2.130.0_4 (active)
  p5.16-error @0.170.230_0 (active)
  p5.16-file-slurp-tiny @0.3.0_0 (active)
  p5.16-gssapi @0.280.0_2 (active)
  p5.16-inc-latest @0.500.0_0 (active)
  p5.16-io-socket-ssl @2.12.0_0 (active)
  p5.16-module-build @0.421.100_0 (active)
  p5.16-module-metadata @1.0.26_0 (active)
  p5.16-net-libidn @0.120.0_4 (active)
  p5.16-net-smtp-ssl @1.10.0_0 (active)
  p5.16-net-ssleay @1.680.0_0 (active)
  p5.16-parse-cpan-meta @1.441.400_0 (active)
  p5.16-pathtools @3.470.0_1 (active)
  p5.16-perl-ostype @1.8.0_0 (active)
  p5.16-pod-escapes @1.70.0_0 (active)
  p5.16-pod-simple @3.290.0_0 (active)
  p5.16-podlators @2.5.3_1 (active)
  p5.16-scalar-list-utils @1.410.0_0 (active)
  p5.16-sub-uplevel @0.250.0_0 (active)
  p5.16-term-readkey @2.320.0_1 (active)
  p5.16-test-exception @0.360.0_0 (active)
  p5.16-test-nowarnings @1.40.0_1 (active)
  p5.16-test-simple @1.1.14_0 (active)
  p5.16-test-warn @0.300.0_0 (active)
  p5.16-tree-dag_node @1.240.0_0 (active)
  p5.16-version @0.991.200_0 (active)
  pcre @8.36_0 (active)
  perl5.16 @5.16.3_1 (active)
  pip_select @0.1_0 (active)
  pkgconfig @0.28_0 (active)
  popt @1.16_0 (active)
  protobuf-cpp @2.5.0_0 (active)
  py27-async-task @0.6.1_0 (active)
  py27-docutils @0.12_0 (active)
  py27-gitdb @0.5.4_0 (active)
  py27-gitpython @0.3.2.RC1_0 (active)
  py27-pip @6.1.1_0 (active)
  py27-protobuf @2.5.0_1 (active)
  py27-roman @2.0.0_0 (active)
  py27-setuptools @15.0_0 (active)
  py27-smmap @0.8.2_0 (active)
  python27 @2.7.9_0 (active)
  python_select @0.3_4 (active)
  rsync @3.1.1_0 (active)
  sqlite3 @3.8.8.3_0 (active)
  vim @7.4.258_0+huge (active)
  xz @5.0.7_0 (active)
  zlib @1.2.8_0 (active)
```

**Comment by psylum** [comment]
Grabbed version 1.1.2 and verified it opens fine on same machine.

**Comment by psequel** [comment]
Fixed in [v1.2.2.beta](http://www.psequel.com/download?version=1.2.2.beta).
